layers/install/load-mos-modules

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+mountpoint /lib/modules && { echo "already mounted"; exit 0; }
+
+mkdir -p /bootkit
+mosctl --debug mount --target=bootkit --dest=/bootkit
+mkdir -p /lib/modules
+mount /bootkit/bootkit/modules.squashfs /lib/modules/
+
+udevadm trigger
```

Comment:
Are we systemd enabled? if so, then maybe

```
systemctl restart systemd-udev-trigger.service
```

if not, this service runs:

```
udevadm trigger --type=subsystems --action=add
udevadm trigger --type=devices --action=add
```
IIUC, `udevadm trigger` will result in:

`udevadm trigger --type=devices --action=change`  which may not create new devices or subsystems